EMS/common-bundle/src/Common/Cluster/SimpleIndexClient.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CommonBundle\Common\Cluster;
+
+use Elastica\Query;
+use EMS\CommonBundle\Common\HttpClientFactory;
+use EMS\Helpers\Html\Headers;
+use EMS\Helpers\Html\MimeTypes;
+use EMS\Helpers\Standard\Json;
+use EMS\Helpers\Standard\Type;
+use GuzzleHttp\Client;
```

Comment:
Use the symfony HttpClient, then you have profiling out of the box.

See \EMS\CommonBundle\Common\CoreApi\CoreApiFactory